themes/src/main/resources-community/theme/base/account/messages/messages_de.properties

```diff
@@ -94,29 +97,43 @@ revoke=Berechtigung widerrufen
 
 configureAuthenticators=Authenticatoren konfigurieren
```

Comment:
I'm happy to change that. However, I would suggest rewording "Authentifikatoren" and "Authenticatoren" to "Mehrfachauthentifizierung" throughout the entire theme. 

It would be strange if we call it "Mehrfachauthentifizierung" when displaying the result of activating or deactivating it, but call it "Authentifikatoren" in the menu and page title. 

TOTP labels are used in the login theme as well. However, it is called "Mobile Authentifizierung" in that theme. 
For the sake of consistency, the same phrase should be used in both themes. Would you prefer "Mobile Authentifizierung" or "Mehrfachauthentifizierung"?

I'll go through the translations of the other themes, because they seem incomplete. Would you prefer that I open another pull-request for hat or update this one?

Reply:
Definitely "Mehrfachauthentifizierung". It refers to to multi-factor-authentication as well as two-factor authentication.

What does "mobile authentication" mean exactly? I understand it as the possibility to configure two-factor authentication, which uses for example Google Authenticator or FreeOTP. If this was the case it should not be called "Mobile Authentication" but "Mehrfachauthentifizierung".

Reply:
Yes, the term is always used in that context. I reworded it to "Mehrfachauthentifizierung" troughout the account theme and the login theme.